Userland/chroot.cpp

```diff
@@ -57,6 +57,8 @@ int main(int argc, char** argv)
                     flags |= MS_NOEXEC;
                 else if (part == "nosuid")
                     flags |= MS_NOSUID;
+                else if (part == "ro")
+                    flags |= MS_RDONLY;
```

Comment:
Just curious, of what use is chrooting read-only?

Reply:
Umm, if you want to run some process(es) in with another directory as the root where they cannot write to that directory? I mean, for the exact same things read-only root and chroot are useful separately.

Reply:
That being said, it's probably not terribly useful for us in practice (in fact, we didn't end up using `chroot()` for isolation at all, because we have `unveil()`) — but it would be weird not to support it if we support other mount flags.

Rather, ask yourself: if you chroot someone to limit what they can access, do you want them to always be able to write to that directory? Sometimes, surely, yes, other times no. Principle of least privilege and all that.

Reply:
I guess it's okay to support :shrug: